kubernetes/schema_volume_source.go

```diff
@@ -90,6 +90,12 @@ func commonVolumeSources() map[string]*schema.Schema {
 						Description: "The URI the data disk in the blob storage",
 						Required:    true,
 					},
+					"kind": {
+						Type:        schema.TypeString,
+						Description: "The type for the data disk. Expected values: Shared, Dedicated, Managed. Defaults to Shared",
+						Optional:    true,
+						Default:     "Shared",
```

Comment:
Setting a default value will cause existing `kubernetes_persistent_volume` resources to show an unexpected diff when users upgrade to the provider release that will contain this change.
To avoid that, the attribute should be marked as `Computed` with no default value set. That way the provider will just pick up on refresh whatever value was set by the API.